src/main/java/service/impl/ParseServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.ParseService;
+import strategy.StrategyService;
+
+public class ParseServiceImpl implements ParseService {
+
+    @Override
+    public Map<String, Integer> parse(List<FruitTransaction> fruitTransactions,
+                                      StrategyService strategyService) {
+        Map<String, Integer> fruitMap = new HashMap<>();
```

Comment:
I think it's better not to store data in this map, but provide every transaction with Handlers over Map in Database.database